.github/workflows/android-maven-release.yml

```diff
@@ -56,11 +83,14 @@ jobs:
           path: .github/aepsdk-commons
```

Comment:
Can you also make similar changes to cache and download scripts from correct branch?